config/middlewares/logging_middleware.py

```
@@ -0,0 +1,43 @@
+import logging
+import time
+
+logger = logging.getLogger(__name__)
+
+
+class LoggingMiddleware:
+    def __init__(self, get_response):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)